.github/workflows/verify-go-src.yml

```diff
@@ -0,0 +1,33 @@
+name: Verify Go client
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  verify-go:
+    name: Verify go client source code is up to date
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go_version: [1.14.x]
```

Comment:
latest is `1.15`, do we want to use that?

Reply:
Happy to do it, but maybe in that case we should also add it to our `Test` action - https://github.com/netlify/open-api/blob/master/.github/workflows/test.yml#L34?